src/parsers/gas.ts

```diff
@@ -133,6 +118,31 @@ export const parseRainbowMeteorologyData = (
     confirmationTimeByPriorityFee,
     currentBaseFee,
   } = rainbowMeterologyData.data;
+
+  // API compatible
+  let blocksToConfirmation: BlocksToConfirmation;
+  if (!rainbowMeterologyData.data.blocksToConfirmation) {
+    const byPriorityFee: BlocksToConfirmationByPriorityFee = {
+      1: confirmationTimeByPriorityFee[15],
+      2: confirmationTimeByPriorityFee[30],
+      3: confirmationTimeByPriorityFee[45],
+      4: confirmationTimeByPriorityFee[60],
+    };
+    const byBaseFee: BlocksToConfirmationByBaseFee = {
```

Comment:
i'll remove this when the change is live on fees estimation API

Reply:
just adding a note that `ConfirmationTimeByPriorityFee` entity will also be removed when u do this